Framework/Algorithms/src/ReflectometryBackgroundSubtraction.cpp

```diff
@@ -270,6 +271,14 @@ void ReflectometryBackgroundSubtraction::exec() {
   }
 
   if (backgroundType == "Polynomial") {
+    int range = static_cast<int>(spectraList.back() - spectraList.front());
+    int degree = getProperty("degreeOfPolynomial");
+    if (range < degree) {
```

Comment:
what if range == degree ?

Reply:
Fitting should work if they are equal as the number of spectra in the list will be 1 greater than the range (e.g if the spectra list is 1, 2 and 3, then the range will be 3 - 1)